README.md

```diff
@@ -34,12 +34,17 @@ So, all in all, it is meant to help the volunteers work more effectively and eff
 
 ## Technology Stack
 
-**Frontend:** HTML, CSS(+ Bootstrap 4), JavaScript  
-**Backend:** Python/Django  
-**Database:** SQL (SQLite3/MySQL/PostgreSQL)  
+### Frontend:   
+<img alt="HTML5" src="https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white"/>    <img alt="CSS3" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white"/>   <img alt="Bootstrap" src="https://img.shields.io/badge/bootstrap%20-%23563D7C.svg?&style=for-the-badge&logo=bootstrap&logoColor=white"/>    <img alt="JavaScript" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E"/>
```

Comment:
Start all `img` tags from a new line.

Reply:
This will make all img one below other

Reply:
No, GitHub interprets two spaces as a line break. If you don't give it those two trailing spaces, it will not ut a line break and all the images will be in-line.

Reply:
Ook

Reply:
It seems like that's not working. No problem, leave it as it is. But one thing I noticed is `</p>` tags without the opening `<p>` tags. Can you remove those?

Also, please squash your commits.

Reply:
Also, provide even spacing between two `img` tags.

Reply:
I have  made it on seperate line now !!

Reply:
Adding a lot of spaces is certainly not a way of putting something in separate lines, Rohan. As I said above, I tried that myself and it was not working, so you can leave that as it was before, just put a single space between all img tags, remove `</p>` tags and squash your commits.